Tasks/NuGetCommand/task.json

```diff
@@ -0,0 +1,447 @@
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613747)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "minimumAgentVersion": "2.115.0",
+    "groups": [
+        {
+            "name": "restoreAuth",
+            "displayName": "Feeds and authentication",
+            "isExpanded": true,
+            "visibleRule": "command = restore"
+        },
+        {
+            "name": "restoreAdvanced",
+            "displayName": "Advanced",
+            "isExpanded": false,
+            "visibleRule": "command = restore"
+        },
+        {
+            "name": "pushAuth",
+            "displayName": "Destination feed and authentication",
+            "isExpanded": true,
+            "visibleRule": "command = push"
+        },
+        {
+            "name": "pushAdvanced",
+            "displayName": "Advanced",
+            "isExpanded": false,
+            "visibleRule": "command = push"
+        },
+        {
+            "name": "packOptions",
+            "displayName": "Pack options",
+            "isExpanded": false,
+            "visibleRule": "command = pack"
+        },
+        {
+            "name": "packAdvanced",
+            "displayName": "Advanced",
+            "isExpanded": false,
+            "visibleRule": "command = pack"
+        }
+    ],
+    "instanceNameFormat": "NuGet $(command)",
+    "inputs": [
+        {
+            "name": "command",
+            "type": "pickList",
+            "label": "Command",
+            "defaultValue": "restore",
+            "helpMarkDown": "The NuGet command to run. Select 'Custom' to add arguments or to use a different command.",
+            "required": "true",
+            "properties": {
+                "EditableOptions": "False"
+            },
+            "options": {
+                "restore": "restore",
+                "pack": "pack",
+                "push": "push",
+                "custom": "custom"
+            }
+        },
+
+        {
+            "name": "solution",
+            "type": "filePath",
+            "label": "Path to solution, packages.config, or project.json",
+            "defaultValue": "**/*.sln",
+            "helpMarkDown": "The path to the solution, packages.config, or project.json file that references the packages to be restored.",
+            "required": "true",
+            "visibleRule": "command = restore"
+        },
+        {
+            "name": "selectOrConfig",
+            "type": "radio",
+            "label": "Feeds to use",
+            "defaultValue": "config",
+            "helpMarkDown": "You can either select feeds from VSTS and/or NuGet.org here, or commit a nuget.config file to your source code repository and set its path here.",
```

Comment:
This makes it sound like I should be able to select multiple.
You can either select a VSTS feed and/or NuGet.org here, ...